website/package.json

```diff
@@ -35,7 +35,7 @@
     "react-dom": "18.2.0",
     "react-hot-toast": "2.4.0",
     "react-search-box": "^3.0.0",
-    "remark-directive": "^3.0.0",
+    "remark-directive": "^2.0.0",
```

Comment:
Had to downgrade this package so it's compatible with Nextra built-in packages for rendering markdown